src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -131,6 +131,10 @@ export class ExposureNotificationService {
       await this.exposureNotification.start();
     } catch (error) {
       captureException('Cannot start EN framework', error);
+      if (error.message === 'ENErrorCodeNotAuthorized (User denied)') {
```

Comment:
I think we can use just the "ENErrorCodeNotAuthorized" part of the message or use the error code.  Preferably the error code if possible.